runners/core-java/src/main/java/org/apache/beam/runners/core/TimerInternals.java

```diff
@@ -225,6 +258,30 @@ public int compareTo(TimerData that) {
     }
   }
 
+  /**
+   * Used for sorting {@link TimerData} by output timestamp of the timer.
```

Comment:
Javadoc that ordering other than the output timestamp is stable but unspecified.